lte/gateway/dev_tools.py

```diff
@@ -137,6 +139,28 @@ def deregister_feg_gw():
     )
 
 
+def check_agw_cloud_connectivity(timeout=10):
+    """
+    Check connectivity of AGW with the cloud using checkin_cli.py
+    Args:
+        timeout: amount of time the command will retry
+    """
+    vagrant_setup("magma", False, force_provision=False)
+    with cd("/home/vagrant/build/python/bin/"):
+        dev_utils.run_fab_command_with_repetition("./checkin_cli.py", timeout)
```

Comment:
Nit here and below: can coalesce these into single line, by specifying the executable's full path

Reply:
Since fab it is a command line automation format, it is better to have each step separated so it is more readable and easier to debug.

It also provides more granularity in case of error (if that folder doesn't exist you will get an explicit error)